src/EnergyPlus/CrossVentMgr.cc

```diff
@@ -1015,6 +1027,25 @@ namespace CrossVentMgr {
 
 	}
 
+	// Clears the global data in MixedAir.
+	// Needed for unit tests, should not be normally called.
+	void
+	clear_state()
+	{
+		HAT_J =  0.0 ;
```

Comment:
HAT_J, HA_J, HAT_R and HA_R are also used in CalcUCSDCV() without being reset to zero so I don't think that it is safe to assume that they are reset always by HcUCSDCV()